src/common/components/organisms/Navigation.tsx

```diff
@@ -202,42 +229,43 @@ const NavItem: React.FC<NavItemProps> = ({ label, active, Icon }) => {
   );
 };
 
-const Navigation = () => {
+const Navigation: React.FC<NavProps> = ({ isEditable, setEditMode }) => {
```

Comment:
I feel like the Navigation shouldn't have a concept of an edit mode, the `Sidebar` should handle placing the edit button onto the side and toggling the view. The Navigation is just there to provide a way to move around, and can be installed somewhere were there is no `editable` to set (and no `setEditMode` function to provide)